examples/basics/scene/mesh_shading.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+
+from vispy import app, scene
+from vispy.io import read_mesh, load_data_file
+from vispy.scene.visuals import Mesh
+from vispy.visuals.filters import ShadingFilter
+
+
+parser = argparse.ArgumentParser()
+default_mesh = load_data_file('orig/triceratops.obj.gz')
+parser.add_argument('--mesh', default=default_mesh)
+parser.add_argument('--shininess', default=None)
+args = parser.parse_args()
+
+vertices, faces, normals, texcoords = read_mesh(args.mesh)
+
+canvas = scene.SceneCanvas(keys='interactive', bgcolor='white')
+view = canvas.central_widget.add_view()
+
+view.camera = 'arcball'
+view.camera.depth_value = 1e3
+
+mesh = Mesh(vertices, faces, color=(.5, .7, .5, 1))
+mesh.set_gl_state('translucent', depth_test=True, cull_face=True)
```

Comment:
Is this needed? The only difference from what the MeshVisual does is `cull_face`, but in the case of this example it doesn't seem to change the resulting visual, at least on my machine.

Reply:
I think it makes a difference with transparency (`alpha < 1` in the color), which I was playing with while testing. So, indeed, this seems redundant.

Reply:
That makes sense. i tried setting the alpha for the mesh to 0.5 and `cull_face=True` mostly did what it should have. I actually like the effect of `cull_face=False` with transparency :wink: 

But anyway, yeah it isn't needed for this. Whoever updates this PR next can remove it.